app/search/searcher.py

```diff
@@ -0,0 +1,40 @@
+from text_query import TextQuery
+import external_search
+import full_text_search
+import indexing
+
+
+class Searcher(object):
+    """A utility class for searching repositories."""
```

Comment:
Maybe "searching person records inside a repository"? Currently it sounds like a class to find a repository.